app/organizations/[slug]/components/OrganizationMembers.tsx

```
@@ -0,0 +1,39 @@
+
+interface OrganizationMembersProps {
+  members: (OrganizationMembership & {
+    person: Person
+  })[]
+  isOwner: boolean
+}
+
+export function OrganizationMembers({ members, isOwner }: OrganizationMembersProps) {
+  const people = members.map(m => m.person)
+  const roles = members.reduce((acc, m) => ({
+    ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)